apps/fluent-tester/macos/Podfile

```diff
@@ -10,7 +10,6 @@ use_test_app! do |target|
 
     # There is a bug where autolinking isn't working, do specify these manually.
     pod 'FRNAvatar', :path => '../../../packages/experimental/Avatar/FRNAvatar.podspec'
-    pod 'FRNCallout', :path => '../../../packages/components/Callout/FRNCallout.podspec'
```

Comment:
Still needed? Could we remove all?

Reply:
can probably be removed wholesale but was trying to stay limited at the time to avoid disruption

Reply:
Can we remove this change then and I'll do it with 0.74?